eureka-client/src/main/java/com/netflix/discovery/CacheRefreshedEvent.java

```diff
@@ -6,6 +6,9 @@
  * 
  * @author brenuart
  */
-public class CacheRefreshedEvent extends DiscoveryEvent {
-
+public class CacheRefreshedEvent extends AbstractEurekaEvent {
```

Comment:
Currently event listeners Depend on DiscoveryEvent <- (CacheRefreshedEvent, StatusChangeEvent) hierarchy. The new API is not backwards compatible, and make break some clients.